sweepbatcher/sweep_batch.go

```diff
@@ -1960,7 +2116,21 @@ func (b *batch) persist(ctx context.Context) error {
 
 // getBatchDestAddr returns the batch's destination address. If the batch
 // has already generated an address then the same one will be returned.
+// The method must not be used in presigned mode. Use getSweepsDestAddr instead.
```

Comment:
should we call this method in the comment `getPresignedSweepsDestAddr` to clarify?

Reply:
Renamed. Thanks!